src/projectscene/qml/Audacity/ProjectScene/clipsview/ClipsSelection.qml

```diff
@@ -58,20 +58,24 @@ Item {
 
         property real startX: 0
         property real startW: 0
+        property bool leftPressed: false
 
         onPressed: function(mouse) {
-            if (mouse.button === Qt.RightButton) {
+            if (mouse.button !== Qt.LeftButton) {
                 return
             }
+            if (!leftPressed) {
+                leftPressed = true
+            }
             leftMa.cursorShape = Qt.ArrowCursor
             leftMa.startX = selRect.x
             leftMa.startW = selRect.width
             leftMa.x = selRect.x
         }
 
         onPositionChanged: function(mouse) {
-            if (mouse.button === Qt.RightButton) {
-                return
+            if (!leftPressed) {
+                return;
```

Comment:
Extra semicolon, I can't get used to not having semicolons :)

Reply:
aaaah :D fixed!